assets/prefabs/Equipment/Diamond/DiamondDagger.java

```diff
@@ -0,0 +1,32 @@
+package teresology;
```

Comment:
Hey, welcome to Terasology!

To add an item, you are going to need `.prefab` file instead of a .java file. A .prefab file is basically a .json file and doesn't need to have anything in line 1-6.

You can see a good example here, https://github.com/Terasology/Equipment/blob/master/assets/prefabs/Equipment/Bronze/BronzeDagger.prefab